components/authentication-framework/org.wso2.carbon.identity.application.authentication.framework/src/main/java/org/wso2/carbon/identity/application/authentication/framework/util/SessionMgtConstants.java

```diff
@@ -35,6 +35,37 @@
     public static final String FEDERATED_AUTHENTICATOR_ID = "AUTHENTICATOR_ID";
     public static final String FEDERATED_PROTOCOL_TYPE = "PROTOCOL_TYPE";
 
+    // filtering constants
```

Comment:
Lets start the comments with an upper case letter and end with a `.`.

Reply:
Comment updated as per suggested.